docs/en/writing-running-appium/caps.md

```diff
@@ -164,8 +164,53 @@ These Capabilities are available only on the [Espresso Driver](/docs/en/drivers/
 
 |Capability|Description|Values|
 |----|-----------|-------|
-|`espressoServerLaunchTimeout`|Timeout in milliseconds used to wait for an espresso server to launch. Defaults to `30000` |e.g., `50000`|
+|`espressoServerLaunchTimeout`|Timeout in milliseconds used to wait for the Espresso server to launch. Defaults to `30000` |e.g., `50000`|
+|`espressoBuildConfig`|Path to the Espresso server build configuration JSON (see below)|e.g., `/projects/myapp-tests/buildconfig.json`|
```

Comment:
I was thinking about a possibility to provide this config as string, because in case of cloud services users don't have direct access to the server machine, they can only provide the caps from the client side

Reply:
That's a good point. Some cloud services allow uploading files to the server machine, but probably not all of them.
My reasoning was that it could make desired capabilities too complex.